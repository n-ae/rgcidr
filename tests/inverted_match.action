-v 192.168.0.0/16 tests/ips.given
