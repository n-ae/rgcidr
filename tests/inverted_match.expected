10.0.0.1
172.16.0.1
8.8.8.8
1.2.3.4
10.0.0.255
172.16.255.254
127.0.0.1
203.0.113.1
198.51.100.1
Not an IP line
Log entry: 10.1.1.1 connected
Error from 172.17.0.1 at timestamp
2001:db8::1
2001:db8:85a3::8a2e:370:7334
::1
fe80::1
